com.qzandronum.Q-Zandronum.yaml

```diff
@@ -0,0 +1,185 @@
+          # Used to be based on Mercurial so that explains the naming
+          - sed -i 's/HG_REVISION_HASH_STRING/"Flatpak Edition"/g' ./src/core/version.cpp
+          - sed -i 's/HG_REVISION_NUMBER/'$(date +%s)'/g' ./src/core/version.cpp
+          - sed -i 's/HG_TIME/"'$(date --iso-8601=seconds)'"/g' ./src/core/version.cpp
+          # Skip irrelevant plugins
+          - sed -i 's/add_opt_subdirectory(chocolate-doom)//g' ./src/plugins/CMakeLists.txt
+          - sed -i 's/add_opt_subdirectory(zandronum)//g' ./src/plugins/CMakeLists.txt
+          - sed -i 's/add_opt_subdirectory(odamex)//g' ./src/plugins/CMakeLists.txt
+          - sed -i 's/add_opt_subdirectory(srb2)//g' ./src/plugins/CMakeLists.txt
+          - sed -i 's/add_opt_subdirectory(turok2ex)//g' ./src/plugins/CMakeLists.txt
+
+  # Zandronum is such a work of art, that it's better for all of us to just
+  # take the upstream package and call it a day.
+  - name: q-zandronum
+    buildsystem: simple
+    build-commands:
+      - tar -xf Q-Zandronum_1.4.5_Linux_amd64.tar.gz
+      - install -D q-zandronum /app/bin/q-zandronum
+      - install -D q-zandronum-server /app/bin/q-zandronum-server
+      - install -D q-zandronum.pk3 /app/share/games/doom/q-zandronum.pk3
+      - install -D libfmodex64-4.44.64.so /app/lib/libfmodex64-4.44.64.so
+    sources:
+      - type: file
+        url: https://github.com/IgeNiaI/Q-Zandronum/releases/download/1.4.5/Q-Zandronum_1.4.5_Linux_amd64.tar.gz
+        sha256: c7c506be92e69be576dcfcc563c4e6fceb5a4b0280bf17948f38a2023c7ef2f4
+          #x-checker-data:
+          #type: html
+          #url: https://zandronum.com/download
+          #version-pattern: <p>The latest version is <strong>([\d\.]+)</strong>, released
+          #  on December 2021.</p>
+          #url-template: https://zandronum.com/downloads/zandronum$version-linux-x86_64.tar.bz2
+
+  - name: launcher
+    buildsystem: simple
+    build-commands:
+      - install -Dm 744 doomseeker.sh /app/bin/doomseeker.sh
+      - install -Dm 644 gzdoom.sf2 /app/share/sounds/sf2/gzdoom.sf2
+      - install -Dm 644 com.qzandronum.Q-Zandronum.desktop -t /app/share/applications
+      - install -Dm 644 com.qzandronum.Q-Zandronum.appdata.xml -t /app/share/metainfo
+      - install -Dm 644 com.qzandronum.Q-Zandronum.48.png /app/share/icons/hicolor/48x48/apps/com.qzandronum.Q-Zandronum.png
+      - install -Dm 644 com.qzandronum.Q-Zandronum.64.png /app/share/icons/hicolor/64x64/apps/com.qzandronum.Q-Zandronum.png
+      - install -Dm 644 com.qzandronum.Q-Zandronum.128.png /app/share/icons/hicolor/128x128/apps/com.qzandronum.Q-Zandronum.png
+    sources:
+      - type: file
+        path: com.qzandronum.Q-Zandronum.desktop
+      - type: file
+        path: com.qzandronum.Q-Zandronum.appdata.xml
+      - type: file
+        path: com.qzandronum.Q-Zandronum.48.png
+      - type: file
+        path: com.qzandronum.Q-Zandronum.64.png
+      - type: file
+        path: com.qzandronum.Q-Zandronum.128.png
```

Comment:
you only need the 128x128. also the other size are lies as they are all the same. Just remove them.